README.md

```diff
@@ -100,7 +109,8 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is responsible for what the variables are in a table, and what data type they will be. You represent a one to many relationship in Active Record by creating ruby class files - a connection file, and model/class files. The model files will either say belongs_to or has_many
+
 ```
```

Comment:
:+1:

A more accurate word to use here in place of "variables" is "columns".

And true, we indicate associations in ActiveRecord using `has_many` and `belongs_to`. But in a schema file, we would indicate the same thing using a foreign key (e.g., `instructor_id`).